### DISCUSSION TITLE
[question] Dispatch setup

### DISCUSSION THREAD

**Discussion by LanceMcCarthy** [mthalman/docker-bump-action#21] [Q&A] [answered]
Thank you @mthalman for this Action!

I've implemented it in [monitor-images.yml](https://github.com/LanceMcCarthy/DevCraftAspire/blob/main/.github/workflows/monitor-images.yml). I was getting some "dredge image compare layers" errors, but it is working fine now ([see last run here](https://github.com/LanceMcCarthy/DevCraftAspire/actions/workflows/monitor-images.yml)). 

However, the downstream workflow [at update-images.yml](https://github.com/LanceMcCarthy/DevCraftAspire/blob/main/.github/workflows/update-images.yml) isn't being triggered by the `base-image-update` repository dispatch. You can see the last runs are all manually triggered by me with workflow_dispatch, none are respository_distach.

Is there a setup step that I missed that enables listening for that name?

In [your demo](https://github.com/mthalman/docker-bump-action-example/blob/main/.github/workflows/monitor-base-image-name.yml), where does the `base-image-update` name come from? I don't see where it is initially set, only where the downstream workflow listens for it.

**Comment by mthalman** [comment, marked as answer]
> Thank you @mthalman for this Action!

Glad to hear someone is using it. It's still a work in progress.

> However, the downstream workflow [at update-images.yml](https://github.com/LanceMcCarthy/DevCraftAspire/blob/main/.github/workflows/update-images.yml) isn't being triggered by the `base-image-update` repository dispatch. You can see the last runs are all manually triggered by me with workflow_dispatch, none are respository_distach.

I only see one run for the monitoring workflow: https://github.com/LanceMcCarthy/DevCraftAspire/actions/runs/9006242681. It indicates that no dispatch was sent because the target image is up-to-date.

> In [your demo](https://github.com/mthalman/docker-bump-action-example/blob/main/.github/workflows/monitor-base-image-name.yml), where does the `base-image-update` name come from? I don't see where it is initially set, only where the downstream workflow listens for it.

`base-image-update` is the default value of the `event-type` action input. See: https://github.com/mthalman/docker-bump-action/blob/549e3efa7acbb1937c2f742761d3d26fe14c4f36/README.md?plain=1#L54 So you can either assume it's that value or set it explicitly to a custom value.

**Reply by LanceMcCarthy** [reply]
OMG... so embarrassing, lol. It indeed wasn't triggered because they are still the same base image.

 That's what I get for trying to do 5 things at once 😂
